src/ape_plugins/_cli.py

```diff
@@ -168,7 +168,40 @@ def install(cli_ctx, skip_confirmation):
             # NOTE: Be *extremely careful* with this command, as it modifies the user's
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
-            subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"])
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"]
+            )
+            if result == 0:
+                cli_ctx.logger.success(f"Plugin '{package_name}' has been added.")
+            else:
+                cli_ctx.logger.error(f"Failed to add '{package_name}'.")
+                sys.exit(1)
+
+
+@cli.command(short_help="Uninstall all plugins in the local config file")
+@ape_cli_context()
+@skip_confirmation_option("Don't ask for confirmation to install the plugins")
+def uninstall(cli_ctx, skip_confirmation):
+    plugins = config.get_config("plugins") or []
```

Comment:
So here, we can just add a `confirmed = click.confirm(f"Uninstall the following plugins? {plugins}")`

And then just pass `-y` into the `subprocess.call`

Reply:
Ah nevermind, let's not do thhis

Reply:
I would need a walkthrough on how to prompt the user and let them pass in a -y